Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -148,7 +148,7 @@ public class PublishPacket: CStruct {
                 messageExpiryInterval: TimeInterval? = nil,
                 topicAlias: UInt16? = nil,
                 responseTopic: String? = nil,
```

Comment:
Not part of this PR but we should double check that we are not making anything a String if it can be non-utf8.